README.md

```diff
@@ -17,6 +26,14 @@ All boolean fields default to null if not specified
 | calendar-resource:resources:list | List all resources | | | |
 | calendar-resource:resource:delete | Delete a resource and anything that belongs to them | `resource_type`* `id`* | | |
 
+### Example for creating a room inspired by the [post from jos](https://help.nextcloud.com/t/22-beta-5-is-here-help-test/118560):
+
+```
+php occ calendar-resource:building:create "Nextcloud office Berlin" "Gerichtstraße 23, 13347 Berlin, Germany" ""
```

Comment:
this information were accidental published. Please change it to some fake data

Reply:
The nextcloud office address isn't secret, but certainly the email addresses should be fake-ified ;-)